cardano-node/src/Cardano/Tracing/Peer.hs

```diff
@@ -0,0 +1,203 @@
+{-# LANGUAGE ConstraintKinds       #-}
+{-# LANGUAGE DeriveAnyClass        #-}
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE DeriveTraversable     #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE OverloadedStrings     #-}
+
+module Cardano.Tracing.Peer
+  ( Peer (..)
+  , NodeKernelData (..)
+  , getCurrentPeers
+  , initialNodeKernelData
+  , ppPeer
+  , setNodeKernel
+  , tracePeers
+  ) where
+
+import           Cardano.Prelude hiding (atomically)
+import           Prelude (String)
+
+import qualified Control.Monad.Class.MonadSTM.Strict as STM
+
+import           Data.Aeson (ToJSON(..), toJSON, Value (..), (.=))
+import           Data.IORef (IORef, readIORef, writeIORef)
+import qualified Data.Map.Strict as Map
+import qualified Data.Set as Set
+import qualified Data.Text as T
```

Comment:
Better to have `import qualified Data.Text as Text`.

Reply:
Fixed.